test/e2e/set_delete_test.go

```diff
@@ -0,0 +1,252 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
+	appsv1 "k8s.io/api/apps/v1"
```

Comment:
Please move the imports to the correct position.

Reply:
yes done